### DISCUSSION TITLE
Flat Surface

### DISCUSSION THREAD

**Discussion by ArtjomPilags** [alicevision/Meshroom#1684] [Q&A]
I have a problem with building a 3D model in Meshroom, I tried it 3 times already with different settings, but the model turns out to be a semicircle, not a flat one.
Do you have any idea how to solve this?
![quest](https://user-images.githubusercontent.com/104992291/166931024-8d8e5693-c809-4800-96e1-dffe5464074e.PNG)
Should look like this
![quest2](https://user-images.githubusercontent.com/104992291/166931273-ca679c39-eb95-4744-9d0d-3cd7eb99f977.PNG)

**Comment by natowi** [comment]
What camera do you use? Is it recognized by Meshroom (green icon)? What kind of lens is being used? Could it be that it is a fisheye lens, but in the camerainit node it is still set to pinhole?
